metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -524,3 +537,57 @@ def main_host_table(since, full_path, until, **kwargs):
     return _copy_table(
         table='main_host', query=f'COPY ({query}) TO STDOUT WITH CSV HEADER', path=full_path, prepend_query=yaml_and_json_parsing_functions()
     )
+
+
+@register('total_workers_vcpu', '1.0', format='json', description=_('Total workers vCPU'), fnc_slicing=limit_slicing)
+def total_workers_vcpu(since, full_path, until, **kwargs):
+    if 'total_workers_vcpu' not in get_optional_collectors():
```

Comment:
This is generating tarball even if it should not. This may be the cause of failing test yesterday. Tarball is empty. This is come from mistake in json collector. We will repair it in follow up PR by ourselves.